executor/runtime/types/types.go

```diff
@@ -446,6 +446,30 @@ func (n *NetworkConfigurationDetails) ToMap() map[string]string {
 	return m
 }
 
+func (n *NetworkConfigurationDetails) ToAnnotationMap() map[string]string {
+	a := map[string]string{}
+	if n.NetworkMode != "" {
+		a[podCommon.AnnotationKeyEffectiveNetworkMode] = n.NetworkMode
+	}
+	if n.IPAddress != "" {
+		a[podCommon.AnnotationKeyIPAddress] = n.IPAddress
+	}
+	if n.EniIPAddress != "" {
+		a[podCommon.AnnotationKeyIPv4Address] = n.EniIPAddress
```

Comment:
Can we rename this annotation? I think it's confusing.

Reply:
Absolutely! 

Did you have a suggestion though? Or do you want me to mimic exactly what comes from the vpc response object?

Reply:
I'm not sure. I think EIP? Or ElasticIPv4Addresss?

Reply:
But why do we have to have the "EIP" part? Sure in the vpcservice we use those words because it is dealing with those api calls, but for pod annotations can we use more generic terms?

Reply:
(that way we could reuse them for other places (ccas, localhost, etc) where they are not EIPs and it won't be weird)